### PR TITLE
feat: react-router-dom implemented

### PR DESCRIPTION
**Pull request by SergioGordillo**

### CODE REVIEW

File: apps/react-app/src/core/routes/routes.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+interface SwitchRoutes {
+  root: string;
+  scene1: string;
+  scene2: string;
```
Comment (brauliodiez) [inline apps/react-app/src/core/routes/routes.ts:4]: Let's change this to the real scenario, we got:

- Home page (paste json)
- Choose template + export


We can name it:

homeScene
TemplateScene

File: apps/react-app/src/pods/screen2/components/screen2.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const Screen2: React.FC = () => {
+  return <h1>Hello from Screen 2</h1>;
+};
```
Comment (brauliodiez) [inline apps/react-app/src/pods/screen2/components/screen2.tsx:5]: You can rename this to the following

src/pods/choose-template/choose-template.container.tsx

And include a barrel index

File: apps/react-app/src/scenes/scene1/scene1.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const Scene1: React.FC = () => {
+  return (
+    <h1>Hello world from scene1</h1>
+  )
+}
```
Comment (brauliodiez) [inline apps/react-app/src/scenes/scene1/scene1.tsx:7]: Same here we can rename this we can name this

inform-json ?

File: apps/react-app/src/core/routes/router.component.tsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { switchRoutes } from './routes';
+import { Scene1 } from '../../scenes/scene1/scene1';
```
Comment (brauliodiez) [inline apps/react-app/src/core/routes/router.component.tsx:4]: manfred-json-input.scene.tsx
template-export.scene.tsx

Comment (brauliodiez) [reply apps/react-app/src/core/routes/router.component.tsx:4]: Use Aliases: https://github.com/Lemoncode/master-frontend-lemoncode/tree/master/04-frameworks/01-react/05-architecture/01-routes

**Review by manudous — APPROVED** [review]

**Review by manudous — COMMENTED** [review]
Resolve the conflicts

**Pull request merged**